fastlane/lib/fastlane/actions/nexus_upload.rb

```diff
@@ -14,7 +14,23 @@ def self.run(params)
       end
 
       def self.upload_url(params)
-        "#{params[:endpoint]}#{params[:mount_path]}/service/local/artifact/maven/content".shellescape
+        url = "#{params[:endpoint]}#{params[:mount_path]}"
+
+        if params[:nexus_version] == 2
+          url << "/service/local/artifact/maven/content"
+        else
+          file_extension = File.expand_path(params[:file]).split('.').last.shellescape
```

Comment:
You should be able to do something like this extend 😊 - `File.extname(params[:file])`

Reply:
Thank you! I will try that.